app/poros/gym.rb

```diff
@@ -1,10 +1,11 @@
 class Gym
-  attr_reader :yelp_gym_id, :name, :address, :phone
+  attr_reader :yelp_gym_id, :name, :address, :address_details, :phone
 
   def initialize(info)
-    @name        = info[:name]
+    @name = info[:name]
     @yelp_gym_id = info[:id]
-    @address     = info[:location][:display_address].join(', ')
-    @phone       = info[:display_phone]
+    @address = info[:location][:display_address].join(', ')
+    @address_details = info[:location].except(:display_address, :cross_streets)
+    @phone = info[:display_phone]
```

Comment:
Interesting one!  I haven't seen `.except` used before!  Neat!

Reply:
Picked it up during RailsEngine when I was handrolling all my serializers 😅